apps/admin/backend/jest.config.js

```diff
@@ -36,4 +36,5 @@ module.exports = {
       '<rootDir>/node_modules/csv-stringify/dist/cjs/sync.cjs',
     '^csv-parse/sync': '<rootDir>/node_modules/csv-parse/dist/cjs/sync.cjs',
   },
+  prettierPath: null,
```

Comment:
@eventualbuddha @kofi-q it looks like jest doesn't support inline snapshots with `prettier@3`. See https://jestjs.io/docs/configuration/#prettierpath-string. It doesn't break existing snapshots, but it prevents generating snapshots or updating existing inline snapshots. Doing this config change turns it off and, in my experience, doesn't introduce any linting problems.

Although I don't add inline snapshots in this PR, I need to update them occasionally and in my test writing will use `toMatchInlineSnapshot` to generate a result for `toEqual`.